### PR TITLE
feat(modem.config): display qr code to config modem

### PR DESCRIPTION
**Pull request by Steffy29**
| Question         | Answer
| ---------------- | ---
| Branch?          | `master`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #UXCT-518
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~Breaking change is mentioned in relevant commits~

## Description

Display QR code for WiFi configuration

### CODE REVIEW

File: packages/manager/apps/telecom/src/app/telecom/pack/xdsl/modem/wifi/config/config.controller.js [hunk 1]
```diff
@@ -243,4 +246,55 @@ export default /* @ngInject */ function XdslModemWifiConfigCtrl(
...
+    const title = $translate.instant('xdsl_modem_wifi_config_modal_title');
+    // Retrieve QR code from APIv6
+    $http
+      .get(
+        `apiv6/xdsl/${$stateParams.serviceName}/modem/wifi/${row.wifiName}/qrCode`,
+      )
+      .then((response) => {
+        // Display modal with the QR code retrieved from APIv6
+        const qrcode = response.data;
+        $uibModal.open({
```
Comment (ghyenne) [inline packages/manager/apps/telecom/src/app/telecom/pack/xdsl/modem/wifi/config/config.controller.js:260]: Good practice is not to use $uibModal which is an old library but in fact a component with template / controller / routing with: 
    views: {
      modal: {
        component: 'xxxxxx',
      },
    },

File: packages/manager/apps/telecom/src/app/telecom/pack/xdsl/modem/wifi/config/modal/wifi-config-modal.html [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+                        >{{'xdsl_modem_wifi_config_modal_ssid' |
+                        translate}}</label
+                    >
+                    {{$ctrl.ssid}}
+                </span>
+                <span>
+                    <label
+                        >{{'xdsl_modem_wifi_config_modal_wifi_name' |
+                        translate}}</label
+                    >
```
Comment (ghyenne) [inline packages/manager/apps/telecom/src/app/telecom/pack/xdsl/modem/wifi/config/modal/wifi-config-modal.html:26]:
```suggestion
                    <label data-translate="xdsl_modem_wifi_config_modal_wifi_name"></label
                    >
```

File: packages/manager/apps/telecom/src/app/telecom/pack/xdsl/modem/wifi/config/modal/wifi-config-modal.html [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+                    {{$ctrl.wifiName}}
```
Comment (ghyenne) [inline packages/manager/apps/telecom/src/app/telecom/pack/xdsl/modem/wifi/config/modal/wifi-config-modal.html:27]:
```suggestion
<span data-ng-bind="$ctrl.wifiName"></span>
```

**Review by ghyenne — DISMISSED** [review]

**Review by antonymarion — APPROVED** [review]

File: packages/manager/apps/telecom/src/app/telecom/pack/xdsl/modem/wifi/config/config.controller.js (hunk 1 above)
Comment (Steffy29) [reply packages/manager/apps/telecom/src/app/telecom/pack/xdsl/modem/wifi/config/config.controller.js]: Change done.

**Review by dectotam — APPROVED** [review]

**Review by antonymarion — APPROVED** [review]

File: packages/manager/apps/telecom/src/app/telecom/pack/xdsl/modem/wifi/config/config.html [hunk 4]
```diff
@@ -98,6 +98,13 @@
                         >
                             <span data-translate="edit"></span>
                         </oui-action-menu-item>
+                        <oui-action-menu-item
+                            on-click="ConfigWifiCtrl.showQRcode($row)"
```
Comment (ghyenne) [inline packages/manager/apps/telecom/src/app/telecom/pack/xdsl/modem/wifi/config/config.html:102]:
```suggestion
                            data-on-click="ConfigWifiCtrl.showQRcode($row)"
```

File: packages/manager/apps/telecom/src/app/telecom/pack/xdsl/modem/wifi/config/index.js [hunk 5]
```diff
@@ -4,6 +4,7 @@ import '@uirouter/angularjs';
 import '@ovh-ux/ng-translate-async-loader';
 import 'angular-translate';
 import 'ovh-api-services';
+import WifiConfigModal from './modal';
```
Comment (anooparveti) [inline packages/manager/apps/telecom/src/app/telecom/pack/xdsl/modem/wifi/config/index.js:7]:
```suggestion
import wifiConfigModal from './modal';
```

File: packages/manager/apps/telecom/src/app/telecom/pack/xdsl/modem/wifi/config/index.js [hunk 6]
```diff
@@ -16,6 +17,7 @@ angular
     'pascalprecht.translate',
     'ovh-api-services',
     'ui.router',
+    WifiConfigModal,
```
Comment (anooparveti) [inline packages/manager/apps/telecom/src/app/telecom/pack/xdsl/modem/wifi/config/index.js:20]:
```suggestion
    wifiConfigModal,
```

**Review by anooparveti — CHANGES_REQUESTED** [review]

**Review by ghyenne — APPROVED** [review]

**Review by antonymarion — APPROVED** [review]

**Review by anooparveti — APPROVED** [review]

**Pull request merged**